src/plugins/intel_gpu/src/kernel_selector/kernels/permute/permute_kernel_bfzyx_to_bfyxz.cpp

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
Sorry for belated comment, but I just realized that `bfzyx -> bfyxz`(tile_8x8_4x4 kernel) is actually equivalent to `bfzyx -> bzyxf` (this kernel). We can just squash some axis.
For example, permute operation of `bfzyx[10, 32, 8, 240, 240] -> bfyxz  [10, 32, 240, 240, 8]` can be converted to `bfzyx[320, 8, 240, 240, 1] -> bzyxf[320, 240, 240, 1, 8]`.
For this transformation, we may 1) add reshape before and after permute or 2) just convert the axis properly before passing it to cl kernel.

How do you think about it? @yeonbok